posthog/models/tag.py

```diff
@@ -0,0 +1,29 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from posthog.models.utils import UUIDModel
+
+
+class Tag(UUIDModel):
+    name: models.CharField = models.CharField(max_length=255)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+
+    class Meta:
+        unique_together = ("name", "team")
+
+    def __str__(self):
+        return self.name
+
+    def clean(self):
+        super().clean()
+        """
+        Ensure that tag doesn't have whitespace
+        """
+        if " " in self.name:
+            raise ValidationError({"name": _(f"Tag name '{self.name}' must not have whitespace.")}, code="invalid")
```

Comment:
Uhm, I think you [misunderstood my comment here](https://github.com/PostHog/posthog/pull/8331#issuecomment-1034332162). Tags should absolutely be able to have spaces inside. 

This line 

> The tag is only made up of letters, numbers, hyphens, and underscores

made it seem like it's not possible to have spaces in tags. If so, I think that's a mistake and we should have spaces.